foundation-extensions/profile/README.md

```diff
@@ -14,7 +14,19 @@ To get started with the Profile extension:
    * Update user attributes.
    * Remove user attributes.
 
-## Add Profile to your App
+## Install the Adobe Experience Platform Profile extension in Experience Platform Launch
+
+1. In Experience Platform Launch, in your mobile property, click the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Profile** extension, and click **Install**.
+3. There are no configuration settings for **Profile**.
+4. Click **Save**.
+5. Follow the publishing process to update SDK configuration.
+
+
+![AEP Profile extension configuration](../../.gitbook/assets/mobile-profile-launch-configuration.png)
```

Comment:
Can you please redo this screenshot, there is a "nfigure Extension" text over the extension icon.